module-3/web-app/src/store/reducers/index.js

```diff
@@ -1,8 +1,10 @@
 import { combineReducers } from "redux"
 import documentReducer from "./documentReducer"
+import documentsReducer from "./documentsReducer"
 
 const reducer = combineReducers({
-  documentReducer
+  documentReducer,
+  documentsReducer
 })
 
 export default reducer
```

Comment:
Didn't we agreed earlier that I shouldn't ever remind of a new line at the end of the file?

Reply:
My bad. Sorry